geomstats/geometry/manifold.py

```diff
@@ -121,6 +139,25 @@ def equip_with_quotient_structure(self):
         self.quotient = self.new(equip=False)
         self.quotient.equip_with_metric(QuotientMetric_, fiber_bundle=self.fiber_bundle)
 
+    def endow_with_math_struct(self, MathStruct, **kwargs):
```

Comment:
rename to `equip_with_structure(self, Structure, **kwargs)` so that it mirrors the metric case. this makes the API easier for users.

Otherwise users have to remember that it's `equip` for metric but `endow` which is cumbersome.